apps/ledger-live-desktop/src/renderer/modals/EditTransaction/steps/StepMethod.tsx

```diff
@@ -0,0 +1,235 @@
+                  <Trans i18nKey={"operation.edit.error.notEnoughFundsToSpeedup"} />
+                ) : (
+                  <Trans i18nKey={"operation.edit.error.notlowestNonceToSpeedup"} />
+                )}
+              </Text>
+            </Description>
+          </Box>
+        </Flex>
+      </EditTypeWrapper>
+      <EditTypeWrapper
+        key={1}
+        selected={isCancel}
+        onClick={() => {
+          if (!disableCancel) {
+            setEditType("cancel");
+          }
+        }}
+      >
+        <Flex flexDirection="row" justifyContent="left" alignItems="center">
+          <CheckBox isChecked={editType === "cancel"} disabled={disableCancel} />
+          <Box>
+            <EditTypeHeader horizontal alignItems="center" selected={isCancel}>
+              <Text fontSize={14} ff="Inter|SemiBold" uppercase ml={1}>
+                <Trans i18nKey={"operation.edit.cancel.title"} />
+              </Text>
+            </EditTypeHeader>
+            <Description selected={isCancel}>
+              <Text ff="Inter|Medium" fontSize={12}>
+                {haveFundToCancel ? (
+                  <Trans
+                    i18nKey={"operation.edit.cancel.description"}
+                    values={{
+                      ticker:
+                        account.type === "TokenAccount"
+                          ? account.token.ticker
+                          : account.currency.ticker,
+                    }}
+                  />
+                ) : (
+                  <Trans i18nKey={"operation.edit.error.notEnoughFundsToCancel"} />
+                )}
+              </Text>
+            </Description>
+          </Box>
+        </Flex>
+      </EditTypeWrapper>
+      <Text
+        ff="Inter|Medium"
+        fontSize={12}
+        style={{ textDecoration: "underline", textAlign: "center", cursor: "pointer" }}
+        onClick={() => openURL(urls.editEthTx.learnMore)}
+      >
+        {t("operation.edit.learnMore")}
+      </Text>
+    </Box>
+  );
+};
+
+export class StepMethodFooter extends PureComponent<StepProps> {
+  render() {
+    const {
+      t,
+      transitionTo,
+      editType,
+      updateTransaction,
+      account,
+      parentAccount,
+      transactionSequenceNumber,
+      isNftOperation,
+      setIsNFTSend,
+      haveFundToSpeedup,
+      haveFundToCancel,
+      isOldestEditableOperation,
+    } = this.props;
+    const transactionRaw = this.props.transactionRaw as EthereumTransactionRaw;
+    if (!account) return null;
+    const bridge = getAccountBridge(account, parentAccount);
+    return (
+      <>
+        <Button
+          id={"send-recipient-continue-button"}
+          primary
+          disabled={(!haveFundToSpeedup || !isOldestEditableOperation) && !haveFundToCancel} // continue button is disable if both "speedup" and "cancel" are not possible
+          onClick={() => {
+            if (isNftOperation) {
+              setIsNFTSend(editType === "speedup");
+            }
```

Comment:
I don't really understand this one ?

Reply:
> As a global remark, I think we agreed with product to verify the transaction being already finalized at each step, didn't we? 🤔

Yes, you are right. Fixed now

Reply:
for the 
"setIsNFTSend(editType === "speedup");"
comment:
because I need to use the "isNFTSend" props to true for the StepConfirmation.tsx
if I am editing/sending a NFT transaction, I need to set isNFTSend=true 
this is why when editType === "speedup" I need "isNFTSend=true"

but for the "cancel", I am just sending 0 Eth to my account, I need "isNFTSend=false"